package.json

```diff
@@ -76,6 +76,12 @@
           "scope": "resource",
           "description": "Collapse parent items by default.",
           "type": "boolean"
+        },
+        "memo.linkBehavior.useLongRefsAlways": {
```

Comment:
One suggestion from is to use enum instead of boolean here, usually boolean flags are less scalable in the long run (We might consider introducing some other mods as well)

How about `memo.links.autocompleteStrategy` with the following options:

- Short links when possible
- Always long links

And with short links enabled by default (as you already did 👍). I think it's a good idea to support both strategies, and I understand that short links might lead to some inconsistencies with duplicates. Still, I think if the user knows how it works, it's rarely a problem. Obsidian uses the same strategy with short links, and most of the users are just fine with it, including me.